packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -1584,6 +1708,24 @@ void main() {
   });
 
   group('constraints', () {
+      testWidgets('default constraints is max width 640 in material 3', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          theme: ThemeData.light(useMaterial3: true),
+          home: const MediaQuery(
+            data: MediaQueryData(size: Size(1000, 1000)),
+            child: Scaffold(
+              body: Center(child: Text('body')),
+              bottomSheet: Placeholder(fallbackWidth: 800),
+            ),
+          ),
+        ),
+      );
+      expect(
```

Comment:
We're checking to see that the bottomsheet expands to its max width, so expected to see a test like:
```dart
expect(tester.getSize(find.byTape(BottomSheet)).width, 640)
```